app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -409,6 +416,14 @@ public void setAudioTrack(int ndx, List<MediaStream> allStreams) {
         }
     }
 
+    public void setPlaybackSpeed(@NonNull Double speed) {
+        if (nativeMode) {
```

Comment:
If you want playbackSpeed to have a default value of `-1` and you think there could be errors caused by that, you could protect against that and log the error here

```suggestion
        if (speed < 0.25f) {
            Timber.d("invalid playback speed");
            return;
        }
        if (nativeMode) {
```

Reply:
I think this is a good safety check we were missing
I've bumped it to warning in this case because if we ever get a value like `-1.0` it means the model controlling playback speed is completely broken too, so users likely won't be able to change their speed either / something is very wrong